src/storage/prisma.ts

```diff
@@ -117,7 +84,18 @@ export class PrismaStorage implements IDriveStorage {
 
     async createDrive(id: string, drive: DocumentDriveStorage): Promise<void> {
         // drive for all drive documents
-        await this.createDocument('drives', id, drive as DocumentStorage);
+        await this.createDocument('drives', id, drive as DocumentStorage, [
+            {
+                syncId: '0',
```

Comment:
For the document drive we use syncId 0 for the global scope and 1 for the local scope. This is not great and we will need to change this when we do the document-centric rework.